react/classes/react-dynamic-data-workshop/src/components/RepoList.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import UserHeader from './UserHeader';
+import { getUserData as UserData } from '../utils/getUserData';
+class RepoList extends Component {
+  state = {
+    repositories: [],
+  };
+  componentDidMount() {
+    const userName = 'bayan-404';
+    UserData(`https://api.github.com/users/${userName}/repos`).then((repos) =>
+      this.setState({ repositories: repos })
+    );
+  }
+  render() {
+    const repos = this.state.repositories;
```

Comment:
here you can use destruction 
```jsx
const {repositories} = this.state; 
```
then you can use it in the next line instead of 
```jsx
  return !this.state.repositories ? (
      <h1>Loading ...</h1>
    ):(
    repos.map((repo) => (....
```
  like this

```jsx
      return !repositories? (
      <h1>Loading ...</h1>
    ) : (
    repositores.map((repo) => (.....
```

Reply:
I'm pretty sure that it's called distraction, not destruction in any way :sunglasses: :woman_dancing:

Reply:
ي عمي انا بدي ادمر العالم 💥

Reply:
and I thought you have a snow-white's heart :broken_heart: :drop_of_blood:

Reply:
wallah ya guys I appreciate you